Client/Frontend/Browser/URIFixup.swift

```diff
@@ -6,6 +6,23 @@ import Foundation
 import Shared
 
 class URIFixup {
+
+    private static func validateURL(_ url: URL) -> URL? {
+        // Validate the domain to make sure it doesn't have any invalid characters
+        // IE: quotes, etc..
+        if let host = url.host {
+            guard let decodedASCIIURL = host.replacingOccurrences(of: "+", with: "").removingPercentEncoding else {
```

Comment:
why is `+` removed?

Reply:
IIRC, it was for email searching as emails could have a plus sign in the host but the function would have returned invalid.. but the code has changed and it works without it.